src/picknik_ur_mujoco_config/objectives/command_vfc.xml

```diff
@@ -13,17 +13,9 @@
             <Action ID="CreateStampedPose" reference_frame="{tip_link}" stamped_pose="{current_pose}"/>
             <Action ID="TransformPoseFrame" output_pose="{current_pose}" input_pose="{current_pose}"/>
             <Action ID="CalculatePoseOffset" source_pose="{initial_pose}" destination_pose="{current_pose}" source_to_destination_pose="{initial_pose_current}"/>
-            <Action ID="UnpackPoseMessage" pose="{initial_pose_current}"/>
```

Comment:
For `main` I think we should revert the changes to this file and the other VFC example Objectives, since the subtree won't work as expected without this.
When we merge into `6.0` we can remove this entire subtree and the two latest VFC examples: `Move Until Contact Example` and `Move With Velocity and Stop Distance`. I think it's okay to have those in the `6.1` release, and they are not needed for training.